openstack/smn/v2/logtank/urls.go

```diff
@@ -0,0 +1,21 @@
+package logtank
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+func createURL(c *golangsdk.ServiceClient, topicUrn string) string {
+	return c.ServiceURL("topics", topicUrn, "logtanks")
+}
+
+func getURL(c *golangsdk.ServiceClient, topicUrn string) string {
+	return c.ServiceURL("topics", topicUrn, "logtanks")
+}
```

Comment:
this two funcs are same, you can merge them into one func, for example, baseURL

Reply:
rename